x-pack/plugins/cloud_security_posture/common/types.ts

```diff
@@ -114,3 +115,22 @@ export interface BenchmarkResponse {
   page: number;
   perPage: number;
 }
+
+export interface GetCspRuleTemplateRequest {
+  search?: string;
+  page: number;
+  perPage: number;
+  fields?: string[];
+  sortField: string;
+  searchFields: string[];
+  sortOrder: string;
+  benchmarkId?: string; // cis_k8s / cis_eks / cis_aws
+  packagePolicyId?: string;
+}
+
+export interface GetCspRuleTemplateResponse {
+  items: CspRuleTemplate[];
+  total: number;
+  page: number;
+  perPage: number;
+}
```

Comment:
I _think_ you can import these from your server schema-config definitions as type only import which should not break any rules:

```ts
import type { TypeOf } from '@kbn/config-schema';
import type { rt } from '../../<your-server-path>';
type GetCspRuleTemplateRequest = TypeOf<typeof rt>;
```

Up to you!